tests/test_ote_training.py

```diff
@@ -446,23 +451,35 @@ class TestOTETraining:
 #                    'person-vehicle-bike-detection-2004'
 #                ],
 #                dataset_name='airport_example',
-#                num_training_iters=None,
 #                usecase='precommit',
 #            ),
-#            TestBunch(
+#            dict(
 #                model_name=[
 #                    'vehicle-detection-0200',
 #                    'vehicle-detection-0201',
 #                    'vehicle-detection-0202',
 #                    'vehicle-detection-0203',
 #                ],
 #                dataset_name='vehicle_detection',
-#                num_training_iters=None,
 #                usecase='precommit',
 #            ),
     ]
 
 
+    @classmethod
+    def _fill_test_parameters_default_values(cl, test_parameters):
```

Comment:
I believe that common convention is to use `cls` for the name of the first argument of a class method. What is this "optimization" introduced for?

Reply:
I supposed that `cl` is a common. Ok, will fix it

Reply:
Fixed